2023-02-14 13:50:32,589 [30444] [ERROR] Error
Traceback (most recent call last):
  File "F:\Projects\ChiaSimplePlotter\plotter.py", line 35, in main
    cache_free = psutil.disk_usage(PLOTTER_PATH)[2]/1024/1024/1024
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\ytx19\AppData\Local\Programs\Python\Python311\Lib\site-packages\psutil\__init__.py", line 2000, in disk_usage
    return _psplatform.disk_usage(path)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\ytx19\AppData\Local\Programs\Python\Python311\Lib\site-packages\psutil\_pswindows.py", line 274, in disk_usage
    total, free = cext.disk_usage(path)
                  ^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [WinError 3] 系统找不到指定的路径。: 'p'
2023-02-14 13:50:37,617 [30444] [ERROR] Error
Traceback (most recent call last):
  File "F:\Projects\ChiaSimplePlotter\plotter.py", line 35, in main
    cache_free = psutil.disk_usage(PLOTTER_PATH)[2]/1024/1024/1024
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\ytx19\AppData\Local\Programs\Python\Python311\Lib\site-packages\psutil\__init__.py", line 2000, in disk_usage
    return _psplatform.disk_usage(path)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\ytx19\AppData\Local\Programs\Python\Python311\Lib\site-packages\psutil\_pswindows.py", line 274, in disk_usage
    total, free = cext.disk_usage(path)
                  ^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [WinError 3] 系统找不到指定的路径。: 'p'
2023-02-14 13:51:43,051 [25228] [ERROR] Error
Traceback (most recent call last):
  File "F:\Projects\ChiaSimplePlotter\plotter.py", line 35, in main
    cache_free = psutil.disk_usage(PLOTTER_PATH)[2]/1024/1024/1024
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\ytx19\AppData\Local\Programs\Python\Python311\Lib\site-packages\psutil\__init__.py", line 2000, in disk_usage
    return _psplatform.disk_usage(path)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\ytx19\AppData\Local\Programs\Python\Python311\Lib\site-packages\psutil\_pswindows.py", line 274, in disk_usage
    total, free = cext.disk_usage(path)
                  ^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [WinError 3] 系统找不到指定的路径。: 'p'
